src/style.css

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+}
+
+.snake {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.snake-part {
+  height: 300px;
+
+  font-family: Arial;
```

In the best, you need to add alternative one, if Arial not found.
```suggestion
  font-family: Arial, Helvetica, sans-serif;
```